splash/render_options.py

```diff
@@ -353,6 +353,12 @@ def get_forbidden_content_types(self):
     def get_html5_media(self):
         return self._get_bool("html5_media", defaults.HTML5_MEDIA_ENABLED)
 
+    def get_engine(self):
+        engine = self.get("engine", default="webkit", type=str)
+        if engine not in {"webkit", "chromium"}:
```

Comment:
I think it's helpful to add "chrome" as an alias to "chromium"

Reply:
hey @lucywang000! I'd prefer not to add "chrome", to avoid confusion; also, technically, engine is not chrome.